src/utils/dialogs.js

```diff
@@ -119,6 +126,81 @@ const RenameDialog = ({ open, onClose, onClick, title, message, currentName }) =
     );
 };
 
+/**
+ * Dialog to export the network case #TODO To be moved in the common-ui repository once it has been created
+ * @param {Boolean} open Is the dialog open ?
+ * @param {EventListener} onClose Event to close the dialog
+ * @param {EventListener} onClick Event to submit the export
+ * @param {String} title Title of the dialog
+ * @param {String} message Message of the dialog
+ */
+const ExportDialog = ({ open, onClose, onClick, title, message, availableFormat}) => {
```

Comment:
I would move it to its own file

Reply:
why this one ? it's not bigger than the others dialog and the file is still small.